### PR TITLE
Update code new icon for MYLUNCH coin

### PR DESCRIPTION
**Pull request by fabriciojd**
Update code new icon for MYLUNCH coin

### CODE REVIEW

**Comment by fabriciojd** [comment]
Please approve this request thanks

File: main.json [hunk 1]
```diff
@@ -127,5 +127,11 @@
       "rootV5": "0:9dd2cd82cbfd74a963be1ef1fd1a1f3e25f60e0b8c742f8b862df378b9a5e265",
       "proxy": "0:9d09ed787c99143c125ecf6b3ace7f6056d6b0180cb69d5a306b8c34dcf0225b"
     }
+    {
+      "symbol": "MYLUNCH",
+      "logoURI": "https://raw.githubusercontent.com/broxus/ton-assets/master/icons/MYLUNCH/logo.svg",
+      "rootV5": "0:e7b16ca2d3cfeae7da962ba0f63d278727ae1f5a9c4a6203816a6ff60cacd6e6",
```
Comment (aleksandr-hramcov) [inline main.json:133]: missed "rootV4"

Comment (fabriciojd) [reply main.json:133]: Please say me where find "rootV4" I'm search in the tonscan plattform but not know where is.

Comment (fabriciojd) [reply main.json:133]: This asset was recently created, I need rootV4?

**Comment by aleksandr-hramcov** [comment]
This is only for upgrading tokens from outdated standard.

**Pull request closed without merging**